CampaignConfig/mdc2020_primary.cfg

```diff
@@ -14,21 +14,18 @@ digidb_version = v1_1
 recodb_version = v1_1
 db_purpose = best
 data_tier = dts
-outdir = /pnfs/mu2e/tape/phy-sim/dts/mu2e/
 stopped = Muminus
 evts = 2000
 primary_name = NoPrimary
```

Comment:
The name of this file is confusing, given that it's for NoPrimary

Reply:
no that is always over ridden, I can try to remove this parameter from the global